### PR TITLE
Update node providers

### PR DESCRIPTION
**Pull request by AnnaNodes**
Hello,

We are a Near node provider NOWNodes.
May you update the node providers list?

### CODE REVIEW

**Comment by bucanero** [comment]
Sorry, but the `http://bit.do/nownodes-near/` URL brings a 404 page.

**Review by bucanero — CHANGES_REQUESTED** [review]
404 URL

File: docs/5.api/rpc/providers.md [hunk 1]
```diff
@@ -20,3 +20,4 @@ balancing.
 | -                                       | Chainstack | https://chainstack.com/build-better-with-near/            |
 | `https://near-mainnet-rpc.allthatnode.com:3030` | All That Node | https://docs.allthatnode.com/protocols/near/          |
 | `https://getblock.io/nodes/near/` | GetBlock | https://getblock.io/nodes/near/         |
+| `http://bit.do/freeapikey-near` | NOWNodes |    http://bit.do/freeapikey-near     |
```
Comment (bucanero) [inline docs/5.api/rpc/providers.md:23]:
```suggestion
| `http://bit.do/freeapikey-near` | NOWNodes |    https://nownodes.io/     |
```

**Review by bucanero — APPROVED** [review]

**Comment by AnnaNodes** [comment]
Hello Damián, 

You were helping me with the NEAR indexer on LinkedIn.

Could you please change the NOWNodes link to https://nownodes.io/nodes?

Thank you!
Anna

**Comment by bucanero** [comment]
> Hello Damián, 
> 
> 
> 
> You were helping me with the NEAR indexer on LinkedIn.
> 
> 
> 
> Could you please change the NOWNodes link to https://nownodes.io/nodes?
> 
> 
> 
> Thank you!
> 
> Anna 

I'm currently OOO but if it's urgent, feel free to open a PR with the new URL and we'll merge it

**Pull request merged**